Small Southern-Cross Edits #8

HawkerTheGreat opened the pull request:
So... theres not much to this, but for now we're done with Southern-Cross map edits. All the issues with launch are fixed here, my issues, your issues, her issues, his issues. I swear... they're all fixed now. If you have any other issues, you're either a lier or you should ping me on the discord asap!

EvilJackCarver commented:
i glanced at #55 earlier and noticed a few miscellaneous files and the backups were deleted. was this intentional? 

i'm not asking you to put 'em back in or anything, i'm just making sure that it was intentional

EvilJackCarver commented:
Gotcha. :+1:

HawkerTheGreat commented:
This will be merged and thrown into the live server, as it is now... I'm fairly certain this is my last commit to this map. I will not be addressing issues related to the sleepers, those will be resolved on the new map with the help of our players, but this map is only temporary. 

Thank you for understanding. I'm trying my best.
